master/definitions.xml

```diff
@@ -16,11 +16,13 @@
       interfaces='SVGAElement'>
     <x:contentmodel xmlns='http://www.w3.org/1999/xhtml'>Descriptive content, plus any element or text allowed by its parent's content model, except for another a element.  If the parent is a switch element, use the content model of the nearest ancestor that isn't a switch.</x:contentmodel>
     <attribute name='href' href='linking.html#AElementHrefAttribute' animatable='yes'/>
-    <attribute name='target' href='linking.html#AElementTargetAttribute' animatable='yes'/>
-    <attribute name='download' href='linking.html#AElementDownloadAttribute' animatable='yes'/>
-    <attribute name='rel' href='linking.html#AElementRelAttribute' animatable='yes'/>
-    <attribute name='hreflang' href='linking.html#AElementHreflangAttribute' animatable='yes'/>
-    <attribute name='type' href='linking.html#AElementTypeAttribute' animatable='yes'/>
+    <attribute name='target' href='linking.html#AElementTargetAttribute' animatable='no'/>
```

Comment:
`target` is animatable in WebKit as far as I can see. This was implemented before the Blink split so I am positive that it is in Blink as well. Could you add the link to the WG resolution if this was resolved on please?

Reply:
target is an SVGAnimatedString in SVG 1.1 and in all browsers including Edge.

The following example works in all browsers, using target.baseVal to read the target.
https://jsfiddle.net/ericwilligers/bL9zvd1w/

Reply:
we agreed to change target to DOMString as the usage is so tiny.

Reply:
Here's the resolution for that decision:
https://github.com/w3c/svgwg/issues/315#issuecomment-376271318